sys/include/net/ng_sixlowpan/frag.h

```diff
@@ -42,10 +42,10 @@ extern "C" {
 #define NG_SIXLOWPAN_FRAG_SIZE_MASK     (0x07ff)    /**< mask for datagram size */
 
 /**
- * @brief   General and 1st 6LoWPAN fragmentation header
+ * @brief   1st 6LoWPAN fragmentation header
  *
- * @see <a href="https://tools.ietf.org/html/rfc4944#section-5.1">
- *          RFC 4944, section 5.1
+ * @see <a href="https://tools.ietf.org/html/rfc4944#section-5.3">
+ *          RFC 4944, section 5.3
```

Comment:
Considering https://github.com/RIOT-OS/RIOT/pull/3305/files#r33813194: 5.1 is the more appropriate section btw.